.travis/install.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+
+# This script was adapted from work by Keith James (keithj). The original source
+# can be found as part of the wtsi-npg/data_handling project here:
+#
+#   https://github.com/wtsi-npg/data_handling
+
+set -e -x
+
+sudo apt-get install libgd2-xpm-dev # For npg_tracking
+sudo apt-get install liblzma-dev # For npg_qc
+sudo apt-get install --yes nodejs
+
+# CPAN
+cpanm --quiet --notest Alien::Tidyp # For npg_tracking
+cpanm --quiet --notest Module::Build
+cpanm --quiet --notest Config::Auto # For npg_qc
+cpanm --quiet --notest Readonly
+
+# WTSI NPG Perl repo dependencies
+cd /tmp
+git clone https://github.com/wtsi-npg/ml_warehouse.git ml_warehouse.git
+git clone https://github.com/wtsi-npg/npg_ml_warehouse.git npg_ml_warehouse.git
+git clone https://github.com/wtsi-npg/npg_tracking.git npg_tracking.git
+git clone https://github.com/wtsi-npg/npg_seq_common.git npg_seq_common.git
+
+cd /tmp/ml_warehouse.git    ; git checkout "${DNAP_WAREHOUSE_VERSION}"
+cd /tmp/npg_ml_warehouse.git; git checkout "${NPG_ML_WAREHOUSE_VERSION}"
+cd /tmp/npg_tracking.git    ; git checkout "${NPG_TRACKING_VERSION}"
+cd /tmp/npg_seq_common.git  ; git checkout "${NPG_SEQ_COMMON_VERSION}"
```

Comment:
I can see some cases where having a easy to find place to move the tag/branch to checkout can be an advantage. But will remove until needed.